internal/cephfs/nodeserver.go

```diff
@@ -173,12 +173,12 @@ func (*NodeServer) mount(ctx context.Context, volOptions *core.VolumeOptions, re
 		case *mounter.FuseMounter:
 			if !csicommon.MountOptionContains(strings.Split(volOptions.FuseMountOptions, ","), readOnly) {
 				volOptions.FuseMountOptions = util.MountOptionsAdd(volOptions.FuseMountOptions, readOnly)
-				fuseMountOptions = append(fuseMountOptions, readOnly)
+				_ = append(fuseMountOptions, readOnly)
```

Comment:
what exactly happening here?

Reply:
actually fuseMoutnOptions and kernelMountOptions are no longer required, so removed now..